### PR TITLE
Feature/r2 168 users history

### PR DESCRIPTION
**Pull request by vaidehiResleeve**
@luca-brg Could you please review this?

### CODE REVIEW

File: invokeai/backend/web/invoke_ai_web_server.py [hunk 1]
```diff
@@ -688,14 +711,6 @@ def handle_request_images(category, earliest_mtime=None):
...
                 )
 
-                if earliest_mtime:
-                    image_paths = list(
-                        filter(
-                            lambda x: os.path.getmtime(x) < earliest_mtime,
-                            image_paths,
-                        )
-                    )
-
```
Comment (luca-brg) [inline invokeai/backend/web/invoke_ai_web_server.py:698]: Was this useful?

**Review by luca-brg — COMMENTED** [review]
Some changes are required

File: invokeai/backend/generate.py [hunk 2]
```diff
@@ -482,32 +482,49 @@ def process_image(image,seed):
...
-        prompt = self.huggingface_concepts_library.replace_concepts_with_triggers(
-            prompt,
-            lambda concepts: self.load_huggingface_concepts(concepts),
-            self.model.textual_inversion_manager.get_all_trigger_strings(),
-        )
+        # prompt = self.huggingface_concepts_library.replace_concepts_with_triggers(
+        #     prompt,
+        #     lambda concepts: self.load_huggingface_concepts(concepts),
+        #     self.model.textual_inversion_manager.get_all_trigger_strings(),
+        # )
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/backend/generate.py:468]: I have removed the commented lines.

File: invokeai/backend/generate.py [hunk 3]
```diff
@@ -865,8 +882,13 @@ def _make_images(
             return None, None
 
         image = self._load_img(img)
+        print("type of image",type(image))
+        print("width of the image",image.width)
+        print(self.width)
+        print("height of the iamge",image.height)
+        print(self.height)
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/backend/generate.py:833]: i am sorry I did not understand your comment.

File: invokeai/backend/generate.py [hunk 4]
```diff
@@ -1281,6 +1303,8 @@ def _fit_image(self, image, max_dimensions):
 
     def _resolution_check(self, width, height, log=False):
         resize_needed = False
+        print(type(width))
+        print(type(height))
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/backend/generate.py]: I have removed the commented lines.

File: invokeai/backend/generator/base.py [hunk 5]
```diff
@@ -279,11 +279,11 @@ class Generator:
     precision: str
     model: DiffusionPipeline
 
-    def __init__(self, model: DiffusionPipeline, precision: str):
-        self.model = model
-        self.precision = precision
+    def __init__(self,):
+        # self.model = model
+        # self.precision = precision
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/backend/generator/base.py]: I have removed the commented lines.

File: invokeai/frontend/CLI/CLI.py [hunk 6]
```diff
@@ -103,8 +103,8 @@ def main():
     # creating a Generate object:
     try:
         gen = Generate(
-            conf=opt.conf,
-            model=opt.model,
+            # conf=opt.conf,
+            # model=opt.model,
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/frontend/CLI/CLI.py]: I have removed the commented lines.

File: invokeai/backend/web/invoke_ai_web_server.py [hunk 7]
```diff
@@ -261,6 +271,7 @@ def upload():
                 )
             if not useSSL:
                 self.socketio.run(app=self.app, host=self.host, port=self.port)
+
```
[the remark this answers is not in the document]
Comment (vaidehiResleeve) [reply invokeai/backend/web/invoke_ai_web_server.py]: I have removed the blank line.

File: invokeai/backend/web/invoke_ai_web_server.py (hunk 1 above)
Comment (vaidehiResleeve) [reply invokeai/backend/web/invoke_ai_web_server.py:698]: It was useful. Good catch. I have added it back.

File: invokeai/backend/generate.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (luca-brg) [reply invokeai/backend/generate.py:833]: Ok, print has been removed

**Pull request merged**